Assets/MixedRealityToolkit/Providers/Hands/HandBounds.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Utility behavior to access the axis aligned bounds of IMixedRealityHands.
+    /// </summary>
+    public class HandBounds : MonoBehaviour, IMixedRealitySourceStateHandler, IMixedRealityHandJointHandler
```

Comment:
After this PR is merged we should talk about incorporating this into IMixedRealityHands.

Reply:
I like that idea! Since you really only need one instance of this.